content/docs/resiliency/release/_index.md

```diff
@@ -13,8 +13,10 @@ Description: >
 
 ### New Features/Changes
 
+- Resiliency protection on driver node pods, see [CSI node failure protection](https://github.com/dell/csm/issues/145).
```

Comment:
The below sentences  need to move from above to here.

- Added support for PowerScale CSI Driver.
- Support for node taint when driver pod is unhealthy

Reply:
I am not sure those are redundant information for sure, I will add the feature link of PowerScale in new feature section

Reply:
@alikdell : Dont need to duplicate , pls move it here